File: CPP/Lambda Function/Largest Number.cpp
class Solution {
public:
    string largestNumber(vector<int>& nums) {
        vector<string>vec;
        for(auto num:nums){
            vec.push_back(to_string(num));
        }
        sort(vec.begin(),vec.end(),[](string &a,string &b){
            return a+b>b+a;
        });
        if(vec[0]=="0")return "0";
        string ans;
        for(auto i:vec){
            ans+=i;
        }
        return ans;
    }
};

File: Python/Lambda Function/Largest Number.py
class Solution:
    def largestNumber(self, nums: List[int]) -> str:
        li=[]
        for num in nums:
            li.append(str(num))
        
        ## Custom function
        def compare(a,b):
            if a+b>b+a:
                return -1
            elif a+b<b+a:
                return 1
            else:
                return 0
        
        li.sort(key=cmp_to_key(compare))

        if li[0]=="0":
            return "0"
        return "".join(li)
